tutorials/exporting_models.ipynb

```diff
@@ -24,12 +24,14 @@
     "import numpy as np\n",
     "from porepy.models.poromechanics import Poromechanics\n",
     "\n",
-    "\n",
-    "params = {\"folder_name\": \"model_exporting\"}\n",
+    "model_params = {\n",
+    "    \"folder_name\": \"model_exporting\"\n",
+    "}\n",
+    "solver_params = {}\n",
     "\n",
     "# The compound class Poromechanics inherits from DataSavingMixin.\n",
-    "model_0 = Poromechanics(params)\n",
-    "pp.run_time_dependent_model(model_0, params)"
+    "model_0 = Poromechanics(model_params)\n",
+    "pp.run_time_dependent_model(model_0, solver_params)"
```

Comment:
Did you include `solver_params` on purpose?

Reply:
This was used before the update on run_model() default argument for param.